tests/data.tf

```diff
@@ -0,0 +1,144 @@
+data "aws_partition" "current" {
+  count = 1
```

Comment:
`count` defaults to 1, so these can be removed.

Reply:
removed